app.rb

```diff
@@ -0,0 +1,121 @@
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rental = []
+  end
+
+  def books_list
+    if @books.empty?
+      puts 'Books list is empty try add a book!'
+    else
+      @books.each_with_index do |book, index|
+        puts "(#{index}) Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+  end
+
+  def people_list
+    if @people.empty?
+      puts 'People list is empty try add a people!'
+    else
+      @people.each do |person|
+        puts "[#{person.class}] ID: #{person.id}, Name: #{person.name}, Age: #{person.age}"
+      end
+    end
+  end
+
+  def show_people
+    @people.each_with_index do |person, index|
+      puts "(#{index}) [#{person.class}] ID: #{person.id}, Name: #{person.name}, Age: #{person.age}"
+    end
+  end
+
+  def show_rental
+    puts 'Select person'
+    show_people
+    print 'Enter selected person ID: '
+    id = gets.chomp.to_i
+    puts 'Rentals:'
+    @rental.each do |rent|
+      puts " Date: #{rent.date} Book: #{rent.book.title} Author: #{rent.book.author}" if rent.person.id == id
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    num = gets.chomp
+    case num
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Invalid input'
+    end
+  end
+
+  def create_student
+    print 'Age:'
+    age = gets.chomp
+    print 'Name:'
+    name = gets.chomp
+    print 'Parent permission [y/n]:'
+    permission = gets.chomp
+    print 'Enter classroom: '
+    classroom = gets.chomp
+    student = Student.new(classroom, age, name, parent_permission: permission)
+    @people.push(student)
+    puts 'Person created successfully!'
+  end
+
```

Comment:
- [ ] When creating a student, the app crashes as soon as I enter the classroom for the student. Kindly fix that so that creating a student works fine as expected. 👍🏽

<img width="1191" alt="Screenshot 2023-10-28 at 1 55 05 PM" src="https://github.com/Rafiduzzaman/Classroom/assets/50941074/1c9267a4-a7d1-48c1-a9d9-51535e73873e">